### PR TITLE
Apachetrademark adjustments

### PR DESCRIPTION
**Pull request by insectengine**
Add the registration mark to first usage of Apache Kafka in the hero and to the footer.

### CODE REVIEW

**Comment by k-wall** [comment]
Do we a statement like "Apache and Apache Kafka are trademarks of the [Apache Software Foundation](https://www.apache.org/)" in the footer?

File: Gemfile.lock [hunk 1]
```diff
@@ -1,23 +1,23 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    addressable (2.8.5)
+    addressable (2.8.6)
```
Comment (k-wall) [inline Gemfile.lock:4]: Do we want these changes in the Gemfile.lock?

Comment (insectengine) [reply Gemfile.lock:4]: Please ignore the changes in the gemfile.lock.

**Comment by insectengine** [comment]
@k-wall  - added the disclaimer to the footer and a few css tweaks for RH logo spacing to adjust for it.

**Review by k-wall — APPROVED** [review]
lgtm

**Comment by k-wall** [comment]
thank you @insectengine

**Pull request merged**